homeassistant/components/evohome/climate.py

```diff
@@ -81,7 +81,7 @@
 
     # evohomeclient has exposed no means of accessing non-default location
     # (i.e. loc_idx > 0) other than using a protected member, such as below
-    tcs_obj_ref = client.locations[loc_idx]._gateways[0]._control_systems[0]  # noqa E501; pylint: disable=protected-access
+    tcs_obj_ref = client.locations[loc_idx]._gateways[0]._control_systems[0]     # noqa: E501; pylint: disable=protected-access
```

Comment:
Why are we changing this line. Doesn't seem needed.

Reply:
I personal affectation - I put lint hints starting column 81 - line 525 is another example.

I feel it makes the code much easier to read - any comment outside col 80 is a lint hint (and stands out) & all other commends end before col 80.

As it goes, almost all lint hints (and ceertainly these two) seem to extend beyond col 79, regardless of where they start.

I will change it if it's a deal-breaker, but would prefer to leave it.

Reply:
Standard is 2 spaces between code and comment.

Reply:
It is two spaces now.